src/native/profiler.cc

```diff
@@ -54,6 +54,7 @@ __thread JNIEnv * Accessors::env_;
 #define MIN_EXP_TIME 5000
 
 #define NUM_CALL_FRAMES 200
+#define MAX_BCI 65536
```

Comment:
Can you add a comment citing where this number comes from? You said it in the commit summary which was ideal, but it's good to encode these things in comments too.

Reply:
Commented. I've also decremented this value to be consistent with its definition.